assets/prototype/domain/shared/RelationsSelector.js

```diff
@@ -2,6 +2,12 @@ import { useEffect, useState } from '@wordpress/element';
 import { Button, MenuItem } from '@blueprintjs/core/lib/esm';
 import { MultiSelect } from '@blueprintjs/select';
 
+const transformRelatedItemsArrayToObject = (array) =>
+	array.reduce((acc, cur) => {
+		acc[cur.id] = cur;
+		return acc;
+	}, {});
```

Comment:
can we rename these vars to give them proper names plz?
I know it's not difficult to understand what is happening here, but it would be better if it was completely effortless.
This just seems out of place within the rest of the code.

Reply:
oops, someone has been writing carelessly code here, my bad

Reply:
Made it more readable here: https://github.com/eventespresso/event-espresso-core/pull/1947/commits/c9c5541900b452363aedb9a8f50296cdbf4347f0